packages/static/src/tabs/tabs-paneless.tsx

```diff
@@ -0,0 +1,31 @@
+import React, {Key} from 'react'
+import TabItem from './item'
+import Tabs, {TabsProps} from './tabs'
+interface ItemProps {
+  key: Key
+  title: string
+}
+
+type TProps = Omit<TabsProps, 'children'>
```

Comment:
'children' shoud go to ItemProps, not TProps. This is the place for html attributes